docs/writing-docs/mdx.md

```diff
@@ -2,6 +2,32 @@
 title: 'MDX'
 ---
 
+- [Basic example](#basic-example)
```

Comment:
@kylegach quick question about this, do we really need this when the structure is replicating what's in the sidebar? The way I see it, we're adding more to a page that is already quite big and will probably continue to grow and could open us up to situations where we inadvertently start introducing some broken links as one might forget to update them, or other similar situations.

Reply:
Oh! 🤦‍♂️ Thanks so much for catching this. We absolutely do not need it. I added it as a helper while writing and accidentally committed it.